### PR TITLE
feat(starterKits): add a logic to grab credentials for starterkits

### PR DESCRIPTION
**Pull request by mamoonraja**

### CODE REVIEW

File: test/test.parse.js [hunk 1]
```diff
@@ -149,4 +153,53 @@ describe('vcap_services', function() {
...
+  });
+
+  it('should get the credentials for Starter from Kube', function() {
+    process.env.service_watson_conversation = JSON.stringify(cloudCredentials);
+    assert.deepEqual(credentials, vcapServices.getCredentialsForStarter('conversation'));
+  });
+
+  it('should get the credentials for Starter from file', function() {
+    assert.deepEqual(credentials, vcapServices.getCredentialsForStarter('conversation', cloudCredentials));
+  });
```
Comment (germanattanasio) [inline test/test.parse.js]: Can you add tests for them things a `null` or `undefined`?
